src/LanguageServer.Engine/Documents/MasterProjectDocument.cs

```diff
@@ -11,6 +11,7 @@
 namespace MSBuildProjectTools.LanguageServer.Documents
 {
     using SemanticModel;
+    using System.Collections.Concurrent;
```

Comment:
Is there a reson you have all `using`s inside a namespace declaration (in general, not only here)? Because the usual .NET convention is to put them on top of the file

Reply:
Ugh that System one was from a merge conflict I think. In some of my projects I use relative namespace imports, I think editor settings may be getting confused (or perhaps I am)…

when I do use them it’s mainly to keep track of which namespaces are from the current (logical) project vs external dependencies.